app/proc/twitter.go

```diff
@@ -12,10 +12,18 @@ import (
 	"github.com/umputun/feed-master/app/feed"
 )
 
-// TwitterClient implements basic publisher of rss itom to twitter
+//go:generate moq -out mocks/tweet_poster.go -pkg mocks -skip-ensure -fmt goimports . TweetPoster
+
+// TweetPoster is the interface for sending messages to telegram
```

Comment:
nit: I think this is a typo, i.e. s/telegram/twitter

Reply:
Fixed the comment: It posts Tweets to Twitter, not Telegram.

Does the name `TweetPoster` look good ?

Reply:
yep, TweetPoster makes sense